docs/setup-robusta/supported-clusters.rst

```diff
@@ -1,96 +1,22 @@
 Supported Clusters
 ################################
 
-We support all Kubernetes distributions other than Minikube.
+Robusta supports all Kubernetes distributions other than Minikube. We tested it on the following clusters
```

Comment:
> We tested it on the following clusters

This seems a little awkward. It implies we tested Robusta just once in the past on these. Perhaps:

> Robusta supports all Kubernetes distributions other than Minikube, including: